Bottom: 72f6448990ccedfa6c82c3332515b5427f37d9df
Top:    2bdefaabf5d24e15a716af2cb3d252ebab9394ee
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-06-22 12:36:55 -0400

Refresh of foobar

---

diff --git a/arch/arm/boot/dts/sun7i-a20.dtsi b/arch/arm/boot/dts/sun7i-a20.dtsi
index 4677fef..30231a9 100644
--- a/arch/arm/boot/dts/sun7i-a20.dtsi
+++ b/arch/arm/boot/dts/sun7i-a20.dtsi
@@ -973,8 +973,8 @@
 			reg-shift = <2>;
 			reg-io-width = <4>;
 			clocks = <&apb1_gates 16>;
-			dmas = <&dma 0 8>, <&dma 0 8>;
-			dma-names = "rx", "tx";
+			/*dmas = <&dma 0 8>, <&dma 0 8>;
+			dma-names = "rx", "tx"; */
 			status = "disabled";
 		};
 
diff --git a/sound/soc/sunxi/sunxi_codec_main.c b/sound/soc/sunxi/sunxi_codec_main.c
index 7733468..00a5f98 100644
--- a/sound/soc/sunxi/sunxi_codec_main.c
+++ b/sound/soc/sunxi/sunxi_codec_main.c
@@ -342,7 +342,7 @@ static int sunxi_codec_probe(struct platform_device *pdev)
 	of_id = of_match_device(snd_sunxi_codec_ids, &pdev->dev);
 	if (!of_id)
 		return -EINVAL;
-#ifdef JDS
+
 	priv = devm_kzalloc(&pdev->dev, sizeof(*priv), GFP_KERNEL);
 	if (!priv) {
 		dev_err(&pdev->dev, "could not allocate DAI object\n");
@@ -413,8 +413,8 @@ static int sunxi_codec_probe(struct platform_device *pdev)
 	ret = snd_soc_register_codec(&pdev->dev, &soc_codec_sunxi_codec, &dit_stub_dai, 1);
 	if (ret)
 		return ret;
-#endif
-//JDS	ret =  devm_sunxi_pcm_platform_register(pdev);
+
+	ret =  devm_sunxi_pcm_platform_register(pdev);
 	printk("JDS - codec driver success registered\n");
 	return ret;
